Jenkinsfile

```diff
@@ -600,6 +656,10 @@ pipeline {
                 }
                 stage('Fp32 OpenCL All gfx1030') {
                     agent{ label rocmnode("navi21") }
+                    options {
+                        timeout(time: 90, unit: 'MINUTES')
```

Comment:
why is this required here?

Reply:
Fp32 OpenCL All gfx1030 recently has some stability issues: ref: https://github.com/ROCmSoftwarePlatform/MIOpen/issues/285

Since it is OCL only and likely not a bug caused by MIOpen changes:
1.  Set a retry(2) for this stage ONLY. because if it failed before, the only way is to restart the whole set of parallel stages, which will take hours and many resources.
2. timeout is for an observation that when this stage is "unstable", usually it will *stuck* after the failure for hours (last observation is that it has ctest failure but stuck for 3 hours). We want it to fail fast and with minimal manual intervention.